src/crypto/index.ts

```diff
@@ -0,0 +1,174 @@
+import { randomBytes } from "crypto"
```

Comment:
Move any imports of packages to either `utils` or you can create a `helpers.ts` in the same directory.